src/oc_erchef/apps/chef_index/c_src/fast_xs.c

```diff
@@ -0,0 +1,210 @@
+#include <assert.h>
```

Comment:
So `chef_index` and `chef_index_batch` are new. Is the batching in `chef_index_batch` largely from the previous `chef_index_expand`, or is that also new?